src/posts/2022-11-30-gsod/index.md

```diff
@@ -0,0 +1,121 @@
+---
+path: '/blog/2022/11/30/cilium-google-season-of-docs-2022'
+date: '2022-11-30T17:00:00.000Z'
+title: 'Cilium Google Season of Docs Case Study'
+ogImage: GSoD.png
+ogSummary: "Learn how Google Season of Docs went for Cilium"
+categories:
+  - Community
+tags:
+  - Cilium
+---
+
+**Cilium Google Season of Docs Case Study: Reorganize Cilium Features and Getting Started**
+
+*Authors: _Bill Mulligan_*
+
+*November 30th, 2022* 
+
+Organization or Project: [Cilium](https://docs.cilium.io/en/v1.12/contributing/development/season_of_docs/)
+
+Organization Description: Cilium (current version 1.11, first released in 2017) is an Apache 2.0-licensed project for providing, securing, and observing network connectivity between container workloads - cloud native, and built around the revolutionary Linux Kernel technology eBPF. Our users adopt Cilium to solve their cloud networking, observability, and security issues at scale including running some of the largest Kubernetes clusters in the world. Cilium has a large community of contributors from over 50 countries around the world. Multiple companies have staff paid to work on Cilium too. Cilium is an incubating project in the Cloud Native Computing Foundation and is the networking data plane for Google Kubernetes Engine and Anthos.
```

Comment:
```suggestion
Organization Description: Cilium (current version 1.12, first released in 2017) is an Apache 2.0-licensed project for providing, securing, and observing network connectivity between container workloads - cloud native, and built around the revolutionary Linux Kernel technology eBPF. Our users adopt Cilium to solve their cloud networking, observability, and security issues at scale including running some of the largest Kubernetes clusters in the world. Cilium has a large community of contributors from over 50 countries around the world. Multiple companies have staff paid to work on Cilium too. Cilium is an incubating project in the Cloud Native Computing Foundation and is the networking data plane for Google Kubernetes Engine and Anthos.
```

Reply:
Are we required to include this? It seems a bit strange on our blog. Could we remove it on the grounds it's a duplicate of what was in the proposal?

Reply:
Yes, it is part of their template https://developers.google.com/season-of-docs/docs/case-study-template